core/types/hashing.go

```diff
@@ -83,7 +83,7 @@ func encodeForDerive(list DerivableList, i int, buf *bytes.Buffer) []byte {
 	return common.CopyBytes(buf.Bytes())
 }
 
-// DeriveSha creates the tree hashes of transactions and receipts in a block header.
+// DeriveSha creates the tree hashes of transactions, receipts, and withdrawals in a block header.
```

Comment:
```suggestion
// DeriveSha creates the tree hashes of a DerivableList like Transactions or Receipts.
```
maybe something more generic instead?